### ISSUE TITLE
[AE2] lasting loss of connection with portions of ME systems due to changes to the network

### ISSUE THREAD

**Issue by Kyrorichekku** [GTNewHorizons/GT-New-Horizons-Modpack#12382]
### Your GTNH Discord Username

kyori#4690

### Your Pack Version

2.2.8

### Your Server

SP

### Type of Server

None

### Your Expectation

Adding cables and misc. other applied energetics 2 devices to the network and the network to recalculate and then continue functioning as normal afterwards (rather vague but its the nature of the problem) 

### The Reality

when adding on to or removing from my network (first built back in early 2.2 and had this same problem back then, even after updates and replacing the controllers did not fix this) anywhere on the network (this is an important distinction) certain outgoing lines (in my case various backbone cables) get cut off becoming inaccessible (while everything else from other cables to the controller still show up)

**Some Notes:**
1. at first this problem did not show up 100% of the time but had a high likelihood of happening but happened more as I added onto my network (but was still very common even early on)

2. my controller and connection to the backbone lines are all within a all in a single chunk and everything is currently loaded when the tests and problems occur

3. the problem does not go away until I either break a part of the controller (what part seems to not matter) OR break a part of line leading to the controller ( and the problem returning when making changes to even other lines)

4. at first it was only one specific backbone (of several) however recently I had started adding storage onto a separate backbone (though this was before updating to 2.2.8) which also started developing the same problem very rarely ( and likewise the increasing likelihood of developing the problem when there are changes to the network but very rare )  leading me to come to the conclusion this a problem seems to have some correlation attached storage

5. since early 2.2 I had only ever seen this occur on lines with some amount storage (with increasing likelihood with more) however since updating to 2.2.8 (and after testing and notably completely removing my controller and replacing it) I observed a similar problem on a backbone dedicated entirely to auto crafting once (despite countless hours of play I'd never seen this happen before ) 

6. there is **some** consistent behavior in when the problem does and does not occur when adding to the network ( and the chance of that problem occurring does not seem directly related to how full the network is) however exactly why is unclear

adding and removing to the network under the same conditions leads to the same behavior of either working or not working (it is not entirely random)

7. the presence or lack of (ME system related) blocks in various configurations on the network results in different behavior for some period of time/similar or until something else changes (with a very unclear logic) that then consistently results in in different behavior (sometimes the problem stops occurring for awhile) 

8. the problem never been observed to be triggered by additions / removals to a non-backbone line connected to the controller (even cases where they are the bridge for the backbone) in my numerous tests

9. **it is very difficult to pin down the when/why exactly it occurs, when I started testing on a miniature replica of my ME system I was able to reliably replicated the problem quite often (see replication section)  however it went from taking a few tries (moving things around was enough) to going hours without being able to replicate the problem in any configuration**

10. one odd behavior I've noticed is whenever the problem would occur (despite having enough channels) this ME drive (in my main network) is visibly shown to be offline despite everything else saying it is connected 
![image](https://user-images.githubusercontent.com/123538122/214560850-51f8b274-d834-46af-883e-e24eb8644eaf.png)

11. since updating from "early 2.2" (but really I mean to the last rc for 2.2.0.0) to 2.2.8 numerous never-before-seen (by me) issues have showed up that may have lead to even more confusing results in testing (but the original issue from early 2.2 is clearly still present and still happens consistently in my own actual ME system)  however I'll be submitting separate bug reports for those

12. I'm available on discord if there are difficulties triggering it in testing (though fair warning I'm a bit brain fried from my own testing)

13. there is more "info" to this however much of it involves what was consistent behavior and then under some conditions contradicting behavior ( as such it was omitted as it is isn't something I can properly explain )   

so for the sake of my own sanity I'm calling this bug report good enough for now and if I figuring anything else out I'll add to it later

**Replication:** 

1. (setup) my testing setup consists of one backbone line with 128 attached storage busses to drawers (this is a simplified version of my own ME system but does not trigger the problem very reliably) 
![setup](https://user-images.githubusercontent.com/123538122/214527936-93eb340c-a274-491e-9264-d9c1387be05b.png)

I have two separate lines sperate from the going out from the controller (one a backbone and one a dense cable) I add/remove interfaces from (these lines are separate from the storage network) for testing 

2. put  some item into the ME system (I recommend adding priorities drawers closer to where you can see)

3.  adding/remove objects from the backbone line may trigger the problem (observable from terminals)

4. descend slowly into madness as it is really annoying to pin down what triggers the problem or even if there is a better way of triggering it 

### Your Proposal

Resolve why resetting the ME cable network completely resolves the problem yet adding/removing to the greater ME system results it sections becoming disconnected

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by CatsLeftEar** [comment]
Yes, it is irritating and can cause unnecessary voiding of materials
